src/coordinate.typ

```diff
@@ -307,8 +307,23 @@
   return t
 }
 
-
-
+/// Resolve a list of coordinates to a absolute vectors
```

Comment:
Typo, either of the following would be fine, I do not know which one is factually correct.
Resolve a list of coordinates to an absolute vector
Resolve a list of coordinates to absolute vectors